### DISCUSSION TITLE
hardware advice

### DISCUSSION THREAD

**Discussion by DariBer** [awawa-dev/HyperHDR#260] [General]
I would like to build my HyperHDR ambient systemt but I have some questions.

I have some source that I connect to my TV, (PS5, AppleTV (gen4), Nintende Switch, VU+ Ultimo 4K cable TV).
I wonder what hardware should I get so that I can connect all these and use it with HyperHDR, what hardware would you recommend that all devices are connected (some 4K, some with HDR and some 1080p), I would like to be able to use HDR & UHE when possible.

Can a good capture card and a HDMI switch (with 4 or 5 inputs) be a good choice? give me advice what I should buy.

thanx
D

**Comment by awawa-dev** [comment]
Hi
HDR2SDR Quality:  https://www.hyperhdr.eu/2022/04/usb-grabbers-hdr-to-sdr-quality-test.html
Grabber latency: https://www.hyperhdr.eu/2021/10/usb-grabbers-grand-latency-test-part-i.html
Can't help with switch.

**Reply by DariBer** [reply]
I guess I will need some kind of unit that makes 1 HDMI source to 2 HDMI outputs (one to the grabber and one to my TV), what's best that support all this 1080p, 4K@60hz and HDR (and no HDR)? :-)

By the way, thank for all help and for your time and investigate to make the software better.

**Comment by awawa-dev** [comment]
You can choose something from Ezcoo or Feintech. Currently I use FeinTech VSP01201 and it works without any problems. I've got also EZ-SP12H2 that is capable of 4kHDR but I can't find confirmation if it's truly 18Gbps device. I don't use any HDCP signal in my setup and sometimes it can make a difference (Ezcoo receives update but they can kill the device, not sure about Feintech and HDCP).

**Comment by DariBer** [comment]
thank for your help!

Do you know anything about this capture card[ ZasLuke USB 3.0 4K
](https://www.amazon.co.uk/gp/product/B08DJ4SXMX/)

**Comment by awawa-dev** [comment]
No, I haven't tested that grabber.
